packages/react-reconciler/src/__tests__/ReactHooksWithNoopRenderer-test.js

```diff
@@ -3867,6 +3867,8 @@ describe('ReactHooksWithNoopRenderer', () => {
     });
   });
 
+  // TODO: delete now that we removed the eager bailout optimization? or do we want to keep this
+  // around in some capacity?
```

Comment:
Yeah could be useful as a way to document the current behavior, or in case we ever add it back. You can retitle to "useReducer does not eagerly bail out of state updates."